Simplenote/Classes/OptionsViewController.swift

```diff
@@ -22,7 +22,7 @@ class OptionsViewController: UIViewController {
 
     /// EntityObserver: Allows us to listen to changes applied to the associated entity
     ///
-    private lazy var entityObserver = EntityObserver(context: SPAppDelegate.shared().managedObjectContext, object: note)
+    private lazy var entityObserver = EntityObserver(context: SPAppDelegate.shared().coreDataManager.managedObjectContext, object: note)
```

Comment:
It seems a bit redundant to access `coreDataManager` only to get `managedObjectContext`. It doesn't seem like we're interested in any other properties of `coreDataManager` outside of `AppDelegate`.
Maybe we can add a shortcut to `AppDelegate` to access `managedObjectContext`, that way we won't need to modify any other files.

Reply:
👍 I was thinking the same thing before.  should be pretty quick and easy to make an app delegate function that returns the moc